.ci.yaml

```diff
@@ -3285,7 +3285,6 @@ targets:
 
   - name: Mac_ios animation_with_microtasks_perf_ios__timeline_summary
     recipe: devicelab/devicelab_drone
-    presubmit: false
```

Comment:
`presubmit: false` changes will not be checked in, this is to validate the devicelab tests in staging.

Reply:
See https://github.com/flutter/flutter/pull/117507/checks?check_run_id=10339070193 for the devicelab tests that ran and passed in the try pool.